src/widgets/mac/qxtglobalshortcut_mac.cpp

```diff
@@ -1,34 +1,34 @@
 #include <Carbon/Carbon.h>
-/****************************************************************************
```

Comment:
What are the changes here?

Reply:
Trailing ```\r\n``` where converted to ```\n```.
VSCode did not show me those changes in the diff, because i had some setting on that hid those type of changes from my diff view.

Reply:
I see; I don't mind switching the EOLs as long as they're consistent at all files.

Reply:
Actually, nvm, let's just go forward with this